toffy/tiling_utils.py

```diff
@@ -377,29 +378,29 @@ def set_tiled_region_params(region_corners_path):
     # store the read in parameters in the region_params key
     tiling_params['region_params'] = generate_region_info(region_params)
 
-    # whether to insert moly points between regions
-    moly_region_insert = read_tiling_param(
-        "Insert a moly point between each tiled region? Y/N: ",
-        "Error: moly point region parameter must be either Y or N",
-        lambda mri: mri in ['Y', 'N', 'y', 'n'],
-        dtype=str
-    )
-
-    # convert to uppercase to standardize
-    moly_region_insert = moly_region_insert.upper()
-    tiling_params['moly_region'] = moly_region_insert
-
-    # whether to insert moly points between fovs
-    moly_interval = read_tiling_param(
-        "Enter the FOV interval size to insert Moly points. If yes, enter the number of FOVs "
-        "between each Moly point. If no, enter 0: ",
-        "Error: moly interval must be 0 or a positive integer",
-        lambda mi: mi >= 0,
-        dtype=int
-    )
-
-    if moly_interval > 0:
-        tiling_params['moly_interval'] = moly_interval
+    # # whether to insert moly points between regions
```

Comment:
I talked to Hadeesha, he wants to leave this in. Can you update the user prompt for each of these to indicate that they'll need to provide the path to the example moly_FOV json file if they want this option?

Reply:
yup yup